docs/samples/Microsoft.ML.Samples/Program.cs

```diff
@@ -1,25 +1,29 @@
 using System;
 using System.Reflection;
+using System.Text.RegularExpressions;
 using Samples.Dynamic;
```

Comment:
Is this required?

Reply:
No, should not be

---
In reply to: [445186042](https://github.com/dotnet/machinelearning/pull/5236#discussion_r445186042) [](ancestors = 445186042)